scripts/lib/k8s.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+ensure_controller_gen() {
+  go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.3.0
+}
+
+ensure_service_controller_running() {
+  echo "Ensure that service controllers are running for given version"
+  # Give executable permission
+  chmod +x ./scripts/generate-crds.sh
```

Comment:
This definitely should not be done in here :) Just change the `scripts/generate-crds.sh` script's execute permission and check in that file to git.